content/hardware/01.mkr/01.boards/mkr-wifi-1010/tutorials/mkr-battery-app-note/mkr-battery-app-note.md

```diff
@@ -0,0 +1,340 @@
+---
+title: 'Understanding Battery Capabilities Of The MKR 1010 WiFi'
+difficulty: intermediate
+description: 'Choose the correct battery for you MKR application.'
+tags:
+  - Battery
+  - MKR
+  - 
+author: 'Ali Jahangiri'
+hardware:
+  - hardware/01.mkr/01.boards/mkr-wifi-1010
+software:
+  - Arduino IDE 2.x
+---
+
+## Introduction
+
+While you can run your MKR board from USB, you can make best use of it when your MKR board is able to operate independently of your computer and a USB cable. This is particularly useful for many IoT applications, use of relays with high current draw and remote sensing. Batteries can help us in this regard and a special connector is included in the MKR products for this end. In this application note, we will take a closer look at the battery capabilities of the MKR boards. 
+
+### Goals
+The goals of this project are to:
+- Understand what the terms `charging capacity` and `charging current`

```

Comment:
Use backticks only for code. When highlighting something like this, let's bold them.

```suggestion
- Understand what the terms **charging capacity** and **charging current** mean.

```

Reply:
Noted!